Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractUse.cs

```diff
@@ -201,6 +203,10 @@ private void DoStartUseObject(object sender, ControllerInteractionEventArgs e)
 
             if (touchedObject != null && interactTouch.IsObjectInteractable(touchedObject))
             {
+                var vrobj = touchedObject.GetComponent<VRTK_InteractableObject>();
```

Comment:
Just keep pushing when you think you've fixed stuff and i'll keep reviewing :)

If you're getting nerves and jitters with git then just pop into the slack channel, i'll help out with commands etc :)